ca.parallel_launcher.ParallelLauncher.json

```diff
@@ -0,0 +1,112 @@
+{
+	"id": "ca.parallel_launcher.ParallelLauncher",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.15-21.08",
+	"sdk": "org.kde.Sdk",
+	"command": "parallel-launcher",
+	"finish-args" : [
+		"--socket=x11",
+		"--socket=wayland",
+		"--socket=pulseaudio",
+		"--share=ipc",
+		"--share=network",
+		"--device=all",
+		"--filesystem=host",
+		"--allow=multiarch",
+		"--talk-name=org.freedesktop.ScreenSaver",
+		"--talk-name=org.freedesktop.PowerManagement.Inhibit",
+		"--talk-name=org.freedesktop.login1",
+		"--filesystem=xdg-run/app/com.discordapp.Discord:create"
+	],
+	"modules": [
+		{
+			"name": "parallel-launcher",
+			"buildsystem": "qmake",
+			"config-opts": [ "CONFIG+=flatpak-version" ],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.com/parallel-launcher/parallel-launcher.git",
+					"commit": "a4c436b0c8077fdbb2ee4aeb68b15d95e7bfb5db"
+				},
+				{
+					"type": "file",
+					"path": "ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+				}
+			],
+			"post-install": [
+				"install -D -t /app/share/parallel-launcher/translations/ lang/*.qm",
+				"mkdir -p /app/share/icons/hicolor/128x128/apps/",
+				"mv -f /app/share/icons/appicon.svg /app/share/icons/hicolor/128x128/apps/ca.parallel_launcher.ParallelLauncher.svg",
+				"mv -f /app/share/mime/packages/bps-mime.xml /app/share/mime/packages/ca.parallel_launcher.ParallelLauncher.xml",
+				"install -D -t /app/share/metainfo/ ca.parallel_launcher.ParallelLauncher.metainfo.xml"
+			]
+		},
+		{
+			"name": "retroarch",
+			"config-opts": [ "--enable-dbus" ],
+			"make-args": [ "GLOBAL_CONFIG_DIR=/app/etc" ],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/libretro/RetroArch.git",
+					"commit": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
+				},
+				{
+					"type": "file",
+					"path": "retroarch.cfg"
+				}
+			],
+			"post-install": [
+				"install -D -t /app/etc/ retroarch.cfg"
+			]
+		},
+		{
+			"name": "retroarch-filers-video",
+			"subdir": "gfx/video_filters",
+			"make-install-args": [ "PREFIX=/app" ],
+			"sources": [{
+				"type": "git",
+				"url": "https://github.com/libretro/RetroArch.git",
+				"commit": "9b282aa742b6c3d2f2925ae5a12e2cd7c6b6ad38"
+			}]
+		},
+		{
+			"name": "retroarch-filters-audio",
+			"subdir": "libretro-common/audio/dsp_filters",
+			"make-install-args": [ "PREFIX=/app" ],
+			"sources": [{
+				"type": "git",
+				"url": "https://github.com/libretro/RetroArch.git",
```

Comment:
this is the exact same source tree as the previous two modules.

Reply:
I believe that's to extract the subdirectory. The RetroArch flatpak does the same thing.

Reply:
Actually, it might be a goof by the RetroArch maintainers. I'll trying removing them and seeing if everything still works.

Reply:
Yeah, turns out it isn't needed at all.